src/components/Table/Table.test.ts

```diff
@@ -93,7 +94,48 @@ describe('sort', () => {
             props: {fields},
```

Comment:
Some comments in these tests that separate the steps would make things much clearer.